typescript/sdk/src/signers/signers.ts

```
@@ -66,6 +65,8 @@ export async function getSignerForChain<TProtocol extends ProtocolType>(
         multiProtocolProvider,
       );
     case ProtocolType.Radix:
+      const { RadixMultiProtocolSignerAdapter } =
+        await import('./radix/radix-toolkit.js');
```

Biome (2.5.5)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)